src/frontend/js/components/CourseRunEnrollment/index.tsx

```diff
@@ -233,12 +235,29 @@ export const CourseRunEnrollment: React.FC<CourseRunEnrollmentProps & CommonData
     case step === Step.ENROLLED:
       return (
         <React.Fragment>
-          <a href={courseRun.resource_link} className="course-run-enrollment__cta">
-            <FormattedMessage {...messages.goToCourse} />
-          </a>
-          <div className="course-run-enrollment__helptext">
-            <FormattedMessage {...messages.enrolled} />
-          </div>
+          {courseRun.starts_in_message ? (
+            <div>
+              {courseRun.dashboard_link ? (
+                <a href={courseRun.dashboard_link} className="course-run-enrollment__cta">
+                  <FormattedMessage {...messages.enrolled} />
+                </a>
+              ) : (
+                <div className="course-run-enrollment__helptext">
+                  <FormattedMessage {...messages.enrolled} />
+                </div>
+              )}
+              <div className="course-run-enrollment__helptext">{courseRun.starts_in_message}</div>
```

Comment:
More semantically correct.
```suggestion
              <p className="course-run-enrollment__helptext">{courseRun.starts_in_message}</div>
```